elytron-oidc-client/src/test/java/org/wildfly/extension/elytron/oidc/Subsystem_3_0_ParsingTestCase.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2023 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.extension.elytron.oidc;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import org.jboss.as.subsystem.test.AbstractSubsystemBaseTest;
+
+/**
+ * Subsystem parsing test case.
+ *
+ * <a href="mailto:fjuma@redhat.com">Farah Juma</a>
+ */
+public class Subsystem_3_0_ParsingTestCase extends AbstractSubsystemBaseTest {
```

Comment:
If you enumerate your subsystem namespaces, you can easily parameterize your parser tests.
e.g.
https://github.com/wildfly/wildfly-core/blob/main/discovery/src/test/java/org/wildfly/extension/discovery/DiscoverySubsystemTestCase.java

Reply:
Hello @pferraro 
Your comments related to parametrizating the tests have been addressed here: https://github.com/wildfly/wildfly/pull/17705